cypress/integration/templates.ts

```diff
@@ -1238,6 +1238,29 @@ context('Template tests', () => {
         });
       });
     });
+
+    it('should validate question template relation input', () => {
+      createTopicWithQuestionsAndRelations();
+
+      cy.login('officer');
+      cy.visit('/');
```

Comment:
In scenarios like this I started using this pattern more: `cy.visit('/ProposalTemplates')`. This is one liner that also saves some time in the test.